recipes-kernel/linux/linux-hikey_4.5.bb

```diff
@@ -0,0 +1,17 @@
+require linux-hikey.inc
+
+DESCRIPTION = "Hisilicon 4.5 Kernel"
+
+PV = "4.5+git${SRCPV}"
+SRCREV_kernel="${AUTOREV}"
```

Comment:
i don't think we want AUTOREV. it creates all kind of issues. the first issue being the inablity to reproduce builds. i will let koen comment more on that..